libbeat/common/jsontransform/jsonhelper.go

```diff
@@ -49,3 +49,7 @@ func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKe
 
 	}
 }
+
+func CreateJSONError(message string) common.MapStr {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function CreateJSONError should have comment or be unexported